CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * When sharing the working copy with a Git repo, we used to forget to export
   branches to Git when only the working copy had changed. That's now fixed.
 
+* Commit description set by `-m`/`--message` is now terminated with a newline
+  character, as well as the one set by editor.
```

Comment:
nit: I parsed the "as well as the one set by edit" as "in addition to the newline set by the editor". That doesn't make any sense, so it should be clear that that's not what it means, but something like "just like descriptions set by editor are" is probably clearer.

Reply:
Updated the changelog as such, thanks.